components/navbar/style.css

```diff
@@ -0,0 +1,81 @@
+header {
+    background-color: #036;
+    border-bottom: 2px solid #fcba19;
+    padding: 0 30px 0 30px;
+    color: #fff;
+    display: flex;
+    height: 65px;
+    top: 0px;
```

Comment:
as a practice, if the value is 0px the 'px' part is usually dropped `0px` -> `0`

Reply:
Fixed.